camera_matrix:
  rows: 3
  cols: 3
  data: [535.4, 0.0, 320.1, 0.0, 539.2, 247.6, 0.0, 0.0, 1.0]
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.425, 0.221, 0.0, 0.0, 0.0]
